File: examples_ios/MyApp/Sources/SurveyLayouts/DefaultLayout/Items/DefaultLayoutLabel.swift
import ConfirmitMobileSDK
import Foundation

enum DefaultLayoutLabelType {
  case text
  case instruction
  case pageTitle
}

class DefaultLayoutLabel: DefaultLayoutItem {
  let value: QuestionText
  let labelType: DefaultLayoutLabelType
  let indent: Int

  init(type: DefaultLayoutLabelType, value: QuestionText, indent: Int = 0) {
    labelType = type
    self.value = value
    self.indent = indent
  }
}

File: examples_android/app/src/main/java/com/example/myapp/layouts/defaultlayout/items/DefaultLayoutLabel.kt
package com.example.myapp.layouts.defaultlayout.items

import com.confirmit.mobilesdk.ui.questions.QuestionText

enum class DefaultLabelType {
    TEXT,
    INSTRUCTION,
    PAGE_TITLE
}

class DefaultLayoutLabel(val labelType: DefaultLabelType, val value: QuestionText, val indent: Int = 0) : DefaultLayoutItem